src/tensor.jl

```diff
@@ -57,7 +57,7 @@ Tensor(sz::Int; dev = -1) = Tensor(Float32, Int(sz), dev = dev)
 
 function at_dim(t::Tensor)
   i = Int32[-1]
-  at_dim(i, t.ptr)
+  Wrapper.at_dim(i, t.ptr)
```

Comment:
This was necessary to avoid a method error - to disambiguate among Wrapper.at_dim and Torch.at_dim.

This could also be solved by renaming Torch.at_dim to something else... - seems like it is only used in L65.

Reply:
IMO with this broader process, nothing in the existing codebase should be sacred. If you want to remove `at_dim` or anything else to make the code cleaner, go for it.

Reply:
Right, good point.

Fixed by renaming/replacing with an implementation of `Base.ndims` (which required a conversion from Int32 to Int).